src/ioctls/vcpu.rs

```diff
@@ -2475,6 +2480,15 @@ mod tests {
         assert!(vcpu.get_reg_list(&mut reg_list).is_ok());
     }
 
+    #[test]
+    fn get_kvm_run() {
```

Comment:
nit: I think we convention is to add `test_` at the beginning of the name.

Reply:
Fixed :)